project_portal_parent_task/views/project_task_portal_template.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <template id="portal_my_task" inherit_id="project.portal_my_task">
+        <xpath expr="//a[@t-field='task.project_id']/.." position="after">
+            <div class="mb8" t-if="task.sudo().parent_id">
+                <strong>Parent Task:</strong>
+                <a t-attf-href="/my/task/#{task.sudo().parent_id.id}" t-field="task.sudo().parent_id.display_name" />
+            </div>
+        </xpath>
+    </template>
```

Comment:
Hello @rivo2302 
We have to remove sudo() from this template, 

and we need to add a new value that check if parent_id exist and user has access to this parent task. 

To do so, we have to inherit "_task_get_page_view_values()" to add a value 'parent_accessible' see exemple [here](https://github.com/odoo/odoo/blob/16.0/addons/project/controllers/portal.py#L243)
and how use it in template [here](https://github.com/odoo/odoo/blob/16.0/addons/project/views/project_portal_templates.xml#L315)